PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBBidRequestExtPrebid.m

```diff
@@ -14,10 +14,19 @@
  */
 
 #import "PBMORTBBidRequestExtPrebid.h"
-#import "PBMORTBAbstract+Protected.h"
 
 @implementation PBMORTBBidRequestExtPrebid : PBMORTBAbstract
 
+- (instancetype)init {
+    if (!(self = [super init])) {
+        return nil;
+    }
+   
+    self.targeting = [PBMMutableJsonDictionary new];
```

Comment:
It's not super clear from the code, just, please make sure that targeting object is absent in the request if it doesn't have any internal properties.

Reply:
Sending this object is required for getting targeting keywords in response. More info [here](https://docs.prebid.org/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#targeting).